### PR TITLE
fix(FileUploader): keyboard accessibility 

### PR DESCRIPTION
**Pull request by hellobrian**
fixes https://github.com/carbon-design-system/carbon-components-react/issues/146

`FileUploader` can now make use of `enter` and `space` keys to trigger opening `FileUploader`

### CODE REVIEW

File: components/FileUploader.js [hunk 1]
```diff
@@ -35,6 +39,11 @@ class FileUploaderButton extends Component {
     }
   }
 
+  handleClick = evt => {
+    console.log('click');
```
Comment (asudoh) [inline components/FileUploader.js]: @hellobrian Is this PR WIP?

Comment (hellobrian) [inline components/FileUploader.js]: It's ready to review now, but looks like I still have to go back and cleanup some things like this! Thank you

**Comment by tw15egan** [comment]
👍 ✅

File: components/FileUploader.js (hunk 1 above)
Comment (asudoh) [inline components/FileUploader.js]: Cool 🙂 The change looks ready to go, but please feel free to have another reviewer. Thanks!

**Comment by chrisdhanaraj** [comment]
Bluemix [toolchain](https://console-regional.ng.bluemix.net/devops/toolchains/6b8b7163-1ec8-4616-82fa-aa467a17af03 "storybook-toolchain-v2"): [Delivery Pipeline](https://console-regional.ng.bluemix.net/devops/pipelines/b84aaac8-8484-4a1e-902a-ebb526eb7998 "storybook-toolchain-v2") deployed [carbon-components-react](https://github.com/carbon-design-system/carbon-components-react/releases/tag/deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170808-205936 "deploy-ibm-yp-us-south-chrisdhanaraj_org-Bluemix-Design-System-20170808-205936") to [Bluemix Design System](https://console.bluemix.net/apps/eadc6714-41e7-421f-8313-02618b55185f?env_id=ibm:yp:us-south "ibm:yp:us-south:chrisdhanaraj_org:Bluemix Design System"), including commit c121ff5e84dba7a65a0be2bfaedc6a8f4c2efe8f

**Pull request merged**